packages/compiler-vapor/__tests__/transforms/__snapshots__/vBind.spec.ts.snap

```diff
@@ -12,6 +12,18 @@ export function render(_ctx) {
 }"
 `;
 
+exports[`compiler v-bind > .attr modifier w/ innerHTML 1`] = `
+"import { setInheritAttrs as _setInheritAttrs, renderEffect as _renderEffect, setAttr as _setAttr, template as _template } from 'vue/vapor';
+const t0 = _template("<div></div>")
+
+export function render(_ctx) {
+  const n0 = t0()
+  _setInheritAttrs(["innerHTML"])
+  _renderEffect(() => _setAttr(n0, "innerHTML", _ctx.foo, true))
```

Comment:
Only `setStyle` and `setClass` needs the finally `root: true` argument due to possible fallthrough attributes - all other helpers should not include this `true` argument.